src/js/common/components/RequiredLoginView.jsx

```diff
@@ -103,7 +103,7 @@ RequiredLoginView.propTypes = {
   authRequired: PropTypes.number.isRequired,
   serviceRequired: PropTypes.string.isRequired,
   userProfile: PropTypes.object.isRequired,
-  loginUrl: PropTypes.string,
+  loginUrl: PropTypes.object.isRequired
```

Comment:
I am actually unclear about when we do/don't require props, but this seems like something we'd want to require either here or in `LoginPrompt` and `VerifyPrompt`. On second thought maybe better there?